### ISSUE TITLE
If client_keys and known_hosts explicitly set to None, task finish with error.

### ISSUE THREAD

**Issue by lynxbyorion** [ronf/asyncssh#27]
In my case, you can not use authorization keys and I tried to turn the off.
``` python
yield from asyncssh.create_connection(self.MySSHClient,
                                              host=node.ip,
                                              username=node.login,
                                              password=node.password,
                                              client_keys=None,
                                              known_hosts=None)
)
```
but
```
ERROR:asyncio:Task exception was never retrieved
future: <Task finished coro=<connect_to_node() done, defined at /var/lib/kali2/kali2/models/manager_nodes.py:41> exception=TypeError("'NoneType' object is not iterable",)>
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/asyncio/tasks.py", line 236, in _step
    result = next(coro)
  File "/var/lib/kali2/kali2/models/manager_nodes.py", line 49, in connect_to_node
    known_hosts=None)
  File "/var/lib/kali2/.env/lib/python3.4/site-packages/asyncssh/connection.py", line 3944, in create_connection
    local_addr=local_addr)
  File "/usr/local/lib/python3.4/asyncio/base_events.py", line 579, in create_connection
    sock, protocol_factory, ssl, server_hostname)
  File "/usr/local/lib/python3.4/asyncio/base_events.py", line 591, in _create_connection_transport
    protocol = protocol_factory()
  File "/var/lib/kali2/.env/lib/python3.4/site-packages/asyncssh/connection.py", line 3932, in conn_factory
    rekey_seconds, auth_waiter)
  File "/var/lib/kali2/.env/lib/python3.4/site-packages/asyncssh/connection.py", line 1710, in __init__
    self._client_keys = _load_private_key_list(client_keys)
  File "/var/lib/kali2/.env/lib/python3.4/site-packages/asyncssh/connection.py", line 206, in _load_private_key_list
    return [_load_private_key(key) for key in keylist]
TypeError: 'NoneType' object is not iterable
```

**Comment by ronf** [comment]
You're absolutely right. This used to work, but it looks like the "None" case got broken back in March when some utility functions were added for loading public/private key lists.

The fix should be simple enough -- I'll try to have something available later tonight.

**Comment by ronf** [comment]
Ok - a fix for this is checked into the "master" and "develop" branches.

I've also released version 1.2.1 to PyPI with this fix and a couple of others minor improvements that were checked into master in the last couple of months.

**Comment by lynxbyorion** [comment]
Thank you.
